boards/components/src/debug_writer.rs

```diff
@@ -31,7 +31,7 @@ use kernel::static_init;
 // least a 1 KiB boundary). This is not _semantically_ critical, but helps keep buffers on 1 KiB
 // boundaries in some cases. Of course, these definitions are only advisory, and individual boards
 // can choose to pass in their own buffers with different lengths.
-const DEBUG_BUFFER_KBYTE: usize = 1;
+const DEBUG_BUFFER_KBYTE: usize = 4;
```

Comment:
Is this just for development, or do we really intend to change this parameter for all boards?

Reply:
Just for development; I will remove.

Reply:
Done.